packages/terra-table/docs/Table.md

```diff
@@ -0,0 +1,40 @@
+# Terra Table
+
+The Terra Table is a structural component to arrange content within a table.
+
+## Getting Started
+
+- Install with [npmjs](https://www.npmjs.com):
+  - `npm install terra-table`
+  - `yarn add terra-table`
+
+## Usage
```

Comment:
We might be able to drop the usage section in favor of the real-code examples in the doc site